code/modules/mob/mob.dm

```diff
@@ -1480,7 +1480,6 @@ Use this proc preferably at the end of an equipment loadout
 				for(var/datum/subsystem/SS in Master.subsystems)
 					SS.stat_entry()
 
-	if(client && client.inactivity < (1200))
```

Comment:
This is very much not equivalent. This appears to make it so that only admins can Alt+Click to view turf contents.

Reply:
oh shoot, I missed client.holder